spider_feeder/loaders.py

```diff
@@ -59,3 +56,37 @@ def _get_store(self, input_uri):
         parsed = urlparse(input_uri)
         store_cls = load_object(self._stores[parsed.scheme])
         return store_cls(input_uri, self._crawler.settings)
+
+    def set_spider_input_data(self, spider, store):
+        raise NotImplementedError()
+
+
+class StartUrlsLoader(BaseLoader):
+    '''Loader setting spider.start_urls. For more information, please refer to BaseLoader.'''
+
+    def set_spider_input_data(self, spider, store):
+        spider.start_urls = [url for (url, _) in store]
```

Comment:
Why aren't you using a generator here? It could use a lot of RAM depending on the size of the input data. 

Is it because of the stats counter? 🤔

Reply:
Nothing special.
Just because I always set `start_urls` as a list
https://docs.scrapy.org/en/latest/topics/spiders.html#scrapy.spiders.Spider.start_urls

Reply:
Also, it looks like there is no constraint for it to a `list`
https://github.com/scrapy/scrapy/blob/master/scrapy/spiders/__init__.py#L69